jenkins_jobs/upload_build_artifacts/script.sh

```diff
@@ -12,16 +12,18 @@ else
 fi
 echo  "{'REPO': '$REPOSITORY_FILE_URL', 'BUILD_TIMESTAMP': '${BUILD_TIMESTAMP}', 'BUILD_LOG': '$BUILD_URL', 'BUILD_ID': $BUILD_JOB_NUMBER, 'BUILD_STATUS': '$BUILD_STATUS'}" > STATUS
 
-alias rsync_upload="rsync -e \
-    'ssh -i ${HOME}/.ssh/${UPLOAD_SERVER_USER_NAME}_id_rsa' \
-    --verbose --compress --stats --times --chmod=a+rwx,g+wx,o-"
+rsync_upload() {
```

Comment:
I think this can be easily deduced from looking at the single command it executes.
Do we really need to add this extra documentation? This was supposed to simplify the script, I can revert back if you think it doesn't.